blocks/edit/da-assets/da-assets.js

```diff
@@ -16,7 +16,13 @@ export async function getRepoId() {
   if (!(repo || owner)) return null;
   const resp = await daFetch(`${DA_ORIGIN}/config/${owner}/${repo}`);
   if (!resp.ok) return null;
-  const json = await resp.json();
+  let json = await resp.json();
+
+  if (json[':type'] === 'multi-sheet') {
+    // If config is a multi-sheet, the data is one level deeper
+    json = json.data;
```

Comment:
Doesn't this mean that the other sheets will be ignored?  Do we need to loop through and parse those sheets to display?